boards/arm/nucleo_h743zi/nucleo_h743zi.dts

```diff
@@ -90,3 +90,9 @@
 &rng {
 	status = "okay";
 };
+
+&spi1 {
+	status = "okay";
+
+	pinctrl-0 = <&spi1_miso_pa6 &spi1_mosi_pb5 &spi1_sck_pa5>;
```

Comment:
Nit, I'd remove the extra line here, just so it aligns with all other ST boards :)

An also it would be nice if we had MOSI on PA7 so the SPI configuration actually aligns with the markings on the board and stays compatible with Arduino headers (D13 SCK (PA5) - D12 MISO (PA7) - D11 MOSI (PA6)).
```suggestion
	status = "okay";
	pinctrl-0 = <&spi1_miso_pa6 &spi1_mosi_pa7 &spi1_sck_pa5>;
```

Then we'll also have to update the doc/index.rst to add the new SPI pins for **Default Zephyr Peripheral Mapping**:
```
- SPI_1 : PA5, PA6, PA7
```